src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,85 @@
+package parser;
+import command.*;
```

Comment:
Avoid using wildcard imports (.*). You can configure import settings in the IDE.